client/app/states/vms/details/_details.sass

```diff
@@ -0,0 +1,25 @@
+.list-view-container
+  &.order-details
+    height: calc(100vh - 272px)
+
+.ss-details-wrapper
+  .order-details
+    .list-view-pf
+      margin-top: 0
+
+$vm-detail-border-color: #d1d1d1;
```

Comment:
So a great time to get familiar with _colors.sass, declaring the hex code here is a nono (furthermore this color already exists) (makes reskinning a MASSIVE *B*) SO check it out, see what the color name should be, (hint `$color...`)